mmdet3d/datasets/transforms/transforms_3d.py

```diff
@@ -2604,26 +2604,29 @@ def laser_mix_transform(self, input_dict: dict, mix_results: dict) -> dict:
         points = input_dict['points']
         pts_semantic_mask = input_dict['pts_semantic_mask']
 
+        # convert angle to radian
+        pitch_angle_down = self.pitch_angles[0] / 180 * np.pi
```

Comment:
Invite @ldkong1205 to take a look.
It seems that the behavior of line 2613 is changed. If so, would we modify the config of Lasermix, so as its docstring?

Reply:
Clarified. Thanks!